src/Model.elm

```diff
@@ -56,6 +59,9 @@ type
     | Unreblog Int
     | Unreblogged (Result Mastodon.Error Mastodon.Status)
     | UserTimeline (Result Mastodon.Error (List Mastodon.Status))
+    | NewWebsocketUserMessage String
```

Comment:
Nit: How about using `WS` instead of `Websocket` for shorter names?

Also and as mentioned in previous PRs I think we're adding too much in `Msg`, we should try moving Mastodon API messages into their own `MastodonMsg` type, with their own `update` fn (a little like how we did for drafts).